Source/EasyNetQ.Hosepipe/ErrorRetry.cs

```diff
@@ -37,24 +38,17 @@ public void RepublishError(Error error, QueueParameters parameters)
             {
                 try
                 {
-                    if (error.Exchange != string.Empty)
-                    {
-                        model.ExchangeDeclarePassive(error.Exchange);
-                    }
-
                     var properties = model.CreateBasicProperties();
                     error.BasicProperties.CopyTo(properties);
-
                     var body = errorMessageSerializer.Deserialize(error.Message);
-
-                    model.BasicPublish(error.Exchange, error.RoutingKey, true, properties, body);
+                    model.BasicPublish("", error.Queue, true, properties, body);
                 }
                 catch (OperationInterruptedException)
                 {
                     Console.WriteLine("The exchange, '{0}', described in the error message does not exist on '{1}', '{2}'",
```

Comment:
Well, I'm not sure that it is possible now to get such an exception :-/

Moreover, I'm 99% sure there will be no check of the existence of the queue.

Probably the best way is to remove this try-catch at all.

Reply:
Come to think about it, you are correct, this cannot be thrown - if the queue does not exist this is a no-op. I remember this from needing to implement my own prioritized RPC queue where the queue to 'respond' to may go away at any time. However if it cannot make it to the exchange, it would be thrown. The insert mechanism has no such try-catch, so we would at least be consistent.

Reply:
Fixed, thanks__